parts/classes/class-script-loader.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/* ---------------------------------------------------------------------------------------------
+   JAVASCRIPT LOADER CLASS
+   Allow `async` and `defer` while enqueueing JavaScript. Based on a solution in WP Rig. 
+   --------------------------------------------------------------------------------------------- */
+
+class TwentyTwenty_Script_Loader {
+
+	/**
+	 * Adds async/defer attributes to enqueued / registered scripts.
+	 *
+	 * If #12009 lands in WordPress, this function can no-op since it would be handled in core.
+	 *
+	 * @link https://core.trac.wordpress.org/ticket/12009
+	 *
+	 * @param string $tag    The script tag.
+	 * @param string $handle The script handle.
+	 * @return string Script HTML string.
+	 */
+	public function filter_script_loader_tag( string $tag, string $handle ) : string {
```

Comment:
Type hinting against `string` argument and return value will only work on PHP7.0, and the minimum requirement (at this point) of PHP in WordPress is still 5.6.

https://mlocati.github.io/articles/php-type-hinting.html

Reply:
Cc @felixarntz and @westonruter

Reply:
I agree, this theme needs to support PHP 5.6

Type hinting will have to wait for the PHP 7.0 bump